README.md

```diff
@@ -22,90 +22,104 @@ In this repo, you'll find the OpenTok React Native library.
 
 ## Installation:
 
-1. In your terminal, change into your React Native project's directory
+1. In your terminal, change into your React Native project's directory.
 
-2. Add the library using `npm` or `yarn`.
+2. Add the library using `npm` or `yarn`:
 
-- `npm install opentok-react-native`
-- `yarn add opentok-react-native`
+  * `npm install opentok-react-native`
+  * `yarn add opentok-react-native`
 
 ### iOS Installation
 
-**Note:** Please make sure to have [CocoaPods](https://cocoapods.org/) on your computer.
-If you've installed this package before, you may need to edit your `Podfile` and project structure because the installation process has changed.
+1. Install the iOS pods:
 
-1. In you terminal, change into the `ios` directory of your React Native project.
+   ```
+   npx pod-install
+   ```
 
-2. Create a pod file by running: `pod init`.
+2. **For React Native versions prior to 0.60**:
 
-**For React Native < 0.60**, add this to your Podfile:
+   * Add this to your Podfile:
 
-```
-    target '<YourProjectName>' do
+     ```
+     target '<YourProjectName>' do
+         # Pods for <YourProject>
+         pod 'OTXCFramework', '2.27.3'
+     end
+     ```
+   
+   * Run `react-native link opentok-react-native`.
 
-      # Pods for <YourProject>
-        pod 'OTXCFramework', '2.27.3'
-    end
+   These steps are not necessary in React Native version 0.60 and later.
 
-```
+3. Ensure you have enabled both camera and microphone usage by adding the following entries to the `Info.plist` file:
 
-3. Now run, `pod install`
+   ```
+   <key>NSCameraUsageDescription</key>
+   <string>Your message to user when the camera is accessed for the first time</string>
+   <key>NSMicrophoneUsageDescription</key>
+   <string>Your message to user when the microphone is accessed for the first time</string>
+   ```
 
-4. After installing the OpenTok iOS SDK, change into your root directory of your project.
+When you create an archive of your app, the [privacy manifest settings required by Apple's App store](https://developer.apple.com/support/third-party-SDK-requirements) are added automatically with this version of the OpenTok React Native SDK.
 
-**For React Native < 0.60**, now run `react-native link opentok-react-native`.
+If you try to archive the app and it fails, please do the following:
 
-5. Open `<YourProjectName>.xcworkspace` contents in XCode. This file can be found in the `ios` folder of your React Native project.
+1. Go to *Target*.
 
-6. Click `File` and `New File`
+2. Click *Build Phases*.
 
-7. Add an empty swift file to your project:
+3. Under the *Link Binary With Libraries* section, remove `libOpenTokReactNative.a` and add it again.
```

Comment:
should we add this step by default and not only if archive fails?

Reply:
This is copied pretty much verbatim from the existing docs (there are just some formatting changes) where it was included under "If you try to archive the app and it fails." So I think we should keep it as is. (It may not always be required.)